### PR TITLE
Migrate to chef 16

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: resources/rdiff_backup.rb [hunk 1]
```diff
@@ -124,6 +124,9 @@
 
   cron new_resource.name do
     user node['rdiff-backup']['server']['user']
+    command ['/usr/bin/flock',
+             node['rdiff-backup']['server']['lock_dir'] + '/' + new_resource.name,
+             filename].join(' ')
```
Comment (ramereth) [inline resources/rdiff_backup.rb:129]: IMHO this is more readable.
```suggestion
    command [
        '/usr/bin/flock',
        node['rdiff-backup']['server']['lock_dir'] + '/' + new_resource.name,
        filename
    ].join(' ')
```

File: Berksfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 source 'https://supermarket.chef.io'
+
+cookbook 'apache2', '< 6.0.0'
```
Comment (ramereth) [inline Berksfile:3]: Please remove this. Not sure how we missed this before.

**Comment by ramereth** [comment]
@TheREK3R please fix the cookstyle issues.

**Comment by ramereth** [comment]
Please update variables.tf to use Chef 16.

**Comment by TheREK3R** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v4.2.0.  Have a nice day!

**Pull request merged**